examples/import_map.json

```diff
@@ -9,6 +9,7 @@
     "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.0.1",
     "std/": "https://deno.land/std@0.159.0/",
     "twind": "https://esm.sh/twind@0.16.17",
-    "twind/": "https://esm.sh/twind@0.16.17/"
+    "twind/": "https://esm.sh/twind@0.16.17/",
+    "chart.js": "https://esm.sh/stable/chart.js@4.0.1/auto"
```

Comment:
```suggestion
    "chart.js": "https://esm.sh/stable/chart.js@4.3.0/auto"
```

Reply:
still unable to bump chart.js to the latest version due to the weird behavior mentioned in https://github.com/denoland/saaskit/pull/338.
i'd do more thorough investigations, potentially on the weekend.
to iterate quickly, i'd stick w/ `4.0.1` for now.

in a follow up pr i'd also take a look at `deps.ts` to bump the version there as well.

Reply:
i was able to bump the version to `4.3.0`, by adding a `jsdelivr` import.  
this begs the question if the underlying issue is caused by esm.sh.

Reply:
the issue could also be caused by fresh itself.
[this example](https://preactjs.com/repl?code=aW1wb3J0IHsgQ2hhcnQgYXMgQ2hhcnRKUywgcmVnaXN0ZXJhYmxlcyB9IGZyb20gImh0dHBzOi8vZXNtLnNoL2NoYXJ0LmpzQDQuMy4wIjsKaW1wb3J0IHsgdXNlRWZmZWN0LCB1c2VSZWYgfSBmcm9tICJwcmVhY3QvaG9va3MiOwoKZXhwb3J0IGZ1bmN0aW9uIHVzZUNoYXJ0KG9wdGlvbnMpIHsKICBjb25zdCBjYW52YXNSZWYgPSB1c2VSZWYobnVsbCk7CiAgY29uc3QgY2hhcnRSZWYgPSB1c2VSZWYobnVsbCk7CgogIHVzZUVmZmVjdCgoKSA9PiB7CiAgICBpZiAoY2FudmFzUmVmLmN1cnJlbnQgPT09IG51bGwpIHsKICAgICAgdGhyb3cgbmV3IEVycm9yKCJjYW52YXMgaXMgbnVsbCIpOwogICAgfQogICAgaWYgKGNoYXJ0UmVmLmN1cnJlbnQpIHsKICAgICAgY2hhcnRSZWYuY3VycmVudC5kZXN0cm95KCk7CiAgICB9CiAgICBDaGFydEpTLnJlZ2lzdGVyKC4uLnJlZ2lzdGVyYWJsZXMpOwoKICAgIGNoYXJ0UmVmLmN1cnJlbnQgPSBuZXcgQ2hhcnRKUygKICAgICAgY2FudmFzUmVmLmN1cnJlbnQsCiAgICAgIG9wdGlvbnMsCiAgICApOwoKICAgIHJldHVybiAoKSA9PiB7CiAgICAgIGNoYXJ0UmVmLmN1cnJlbnQ%2FLmRlc3Ryb3koKTsKICAgIH07CiAgfSwgW2NhbnZhc1JlZiwgb3B0aW9uc10pOwoKICByZXR1cm4geyBjYW52YXNSZWYsIGNoYXJ0UmVmIH07Cn0KCmNvbnN0IG9wdGlvbnM9IHsKICB0eXBlOiAiYmFyIiwKICBkYXRhOiB7CiAgICBsYWJlbHM6IFsiMSIsICIyIiwgIjMiXSwKICAgIGRhdGFzZXRzOiBbCgkJICB7CiAgICAgICAgbGFiZWw6ICJTZXNzaW9ucyIsCiAgICAgICAgZGF0YTogWzEyMywgMjM0LCAyMzRdLAogICAgICAgIGJvcmRlcldpZHRoOiAxLAogICAgICB9LAoJCSAgewogICAgICAgIGxhYmVsOiAiVXNlcnMiLAogICAgICAgIGRhdGE6IFszNDYsIDIzMywgMTIzXSwKCiAgICAgICAgYm9yZGVyV2lkdGg6IDEsCiAgICAgIH0KCQldLAogIH0sCn07CgpleHBvcnQgZnVuY3Rpb24gQ2hhcnQoeyBvcHRpb25zIH0pIHsKICBjb25zdCB7IGNhbnZhc1JlZiB9ID0gdXNlQ2hhcnQob3B0aW9ucykKICByZXR1cm4gPGNhbnZhcyByZWY9e2NhbnZhc1JlZn0gLz47Cn0KCnJlbmRlcig8Q2hhcnQgb3B0aW9ucz17b3B0aW9uc30gLz4sIGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKCdhcHAnKSk7Cg%3D%3D) w/ an esm.sh import works perfectly fine in the preact repl.

Reply:
as discovered by @iuioiua in https://github.com/denoland/saaskit/pull/357, it was just a matter of using the wrong target.